modules/op-release-notes-1-0.adoc

```diff
@@ -0,0 +1,88 @@
+* Support for authentication and authorization at the EventListener Trigger level.
+
+== Deprecated features
+The following items are deprecated or not supported in release 1.0:
+
+* Environment variable `$HOME`, and variable `workingDir` of Steps specification will be changed in the next release. Currently in a Step container, `HOME` and  `workingDir` are overwritten to `/tekton/home` and `/workspace` respectively.
++
+From next release onwards, these two fields will not be modified, and will be set to values defined in the container image and Task's YAML.
+For this release, use flags `disable-home-env-overwrite` and `disable-working-directory-overwrite` to disable overwrite of the `HOME` and `workingDir` variables.
+
+* The following commands are deprecated and will be removed in next release:
+** `tkn pipeline create`
+** `tkn task create`
+
+* The `-f` flag with the `tkn resource create` command is now deprecated. It will be removed in the next release.
+
+* The `-t` flag and the `--timeout` flag ( with seconds format) for the `tkn clustertask create` command are now deprecated. Only duration timeout format is now supported, for example `1h30s`. These deprecated flags will be removed in the next release.
+
+[id="known-issues-1-4-0_{context}"]
+== Known issues
+* If you are upgrading from an older version of {pipelines-title}, you will need to delete your existing deployments before upgrading to {pipelines-title} version 1.0.
+* Submitting the same `v1alpha1` Tasks more than once will result in an error. Use `oc replace` instead of `oc apply` when re-submitting a `v1alpha1` Task.
+
+[id="fixed-issues-1-0_{context}"]
+== Red Hat Fixed issues
+* link:https://issues.redhat.com/browse/SRVKP-637[SRVKP-637] : Previously, the `DeploymentConfig` Task triggered a new deployment build even when an image build was already in progress. This caused the deployment in Pipeline to fail. With this fix, the `deploy task` command  is now replaced with the `oc rollout status` command which waits for the in-progress deployment to finish.
+* link:https://issues.redhat.com/browse/SRVKP-631[SRVKP-631]: Support for `APP_NAME` parameter is now added in Pipeline templates.
+* link:https://issues.redhat.com/browse/SRVKP-616[SRVKP-616]: Pipeline template for Java S2I failed to look up image in the registry. With this fix, the look up will now be performed using the existing image Pipeline Resources instead of user provided `IMAGE_NAME` parameter.
+* link:https://issues.redhat.com/browse/SRVKP-383[SRVKP-383]: All the Openshift Pipelines images are now based on the Red Hat Universal Base Images (UBI).
+* link:https://issues.redhat.com/browse/SRVKP-700[SRVKP-700]:  Previously, `tkn version` command displayed the Pipeline version as `unknown` when the Pipeline is installed in the namespace other than `tekton-pipelines`. With this fix, `tkn version` command now displays the correct Pipeline version in any namespace.
```

Comment:
Previously, when a Pipeline was installed in a namespace other than `tekton-pipelines`, the `tkn version` command displayed the Pipeline version as `unknown`.
...fix, the `tkn version`

Reply:
Previously, when a Pipeline